src/runtime/device_interface.h

```diff
@@ -1,8 +1,10 @@
 #ifndef HALIDE_RUNTIME_DEVICE_INTERFACE_H
 #define HALIDE_RUNTIME_DEVICE_INTERFACE_H
 
+#include "HalideRuntime.h"
+
 #ifndef WEAK
```

Comment:
Can cut the ifndef entirely if including HalideRuntime.h

Reply:
done